coverage/.resultset.json

```
@@ -0,0 +1,437 @@
+          1,
+          1,
+          null,
+          null,
+          null,
+          null,
+          null,
+          null,
+          null,
+          null,
+          null,
+          null,
+          1,
+          0,
+          null,
+          null,
+          null,
+          null,
+          null,
+          null,
+          null,
+          null,
+          null,
+          null,
+          1,
+          0,
+          null,
+          null,
+          null,
+          null,
+          null,
+          null,
+          null,
+          null,
+          null,
+          null,
+          1,
+          0,
+          null,
+          null,
+          null,
+          null,
+          null,
+          null,
+          null,
+          null,
+          null,
+          null,
+          null,
+          null,
+          null,
+          null,
+          null,
+          null,
+          1,
+          null,
+          0,
+          0,
+          null,
+          0,
+          0,
+          null,
+          0,
+          null,
+          null,
+          null
+        ],
+        "branches": {
+          "[:if, 0, 83, 6, 83, 99]": {
+            "[:then, 1, 83, 6, 83, 71]": 2,
+            "[:else, 2, 83, 6, 83, 99]": 0
+          },
+          "[:if, 3, 95, 6, 95, 91]": {
+            "[:then, 4, 95, 6, 95, 59]": 0,
+            "[:else, 5, 95, 6, 95, 91]": 0
+          },
+          "[:if, 6, 125, 14, 128, 17]": {
+            "[:then, 7, 126, 16, 127, 44]": 2,
+            "[:else, 8, 125, 14, 128, 17]": 2
+          },
+          "[:if, 9, 132, 8, 135, 11]": {
+            "[:then, 10, 133, 10, 134, 28]": 1,
+            "[:else, 11, 132, 8, 135, 11]": 0
+          },
+          "[:if, 12, 174, 8, 174, 81]": {
+            "[:then, 13, 174, 8, 174, 56]": 1,
+            "[:else, 14, 174, 8, 174, 81]": 22
+          },
+          "[:if, 15, 175, 8, 178, 11]": {
+            "[:then, 16, 176, 10, 177, 13]": 4,
+            "[:else, 17, 175, 8, 178, 11]": 19
+          },
+          "[:if, 18, 230, 6, 233, 58]": {
+            "[:then, 19, 231, 8, 231, 65]": 0,
+            "[:else, 20, 233, 8, 233, 58]": 0
+          },
+          "[:if, 21, 228, 6, 234, 9]": {
+            "[:then, 22, 229, 8, 229, 65]": 1,
+            "[:else, 23, 230, 6, 233, 58]": 0
+          },
+          "[:unless, 24, 246, 6, 246, 104]": {
+            "[:else, 25, 246, 6, 246, 104]": 1,
+            "[:then, 26, 246, 6, 246, 75]": 1
+          },
+          "[:unless, 27, 248, 6, 248, 117]": {
+            "[:else, 28, 248, 6, 248, 117]": 0,
+            "[:then, 29, 248, 6, 248, 83]": 1
+          },
+          "[:if, 30, 358, 20, 358, 84]": {
+            "[:then, 31, 358, 28, 358, 43]": 0,
+            "[:else, 32, 358, 47, 358, 83]": 0
+          },
+          "[:if, 33, 359, 6, 359, 57]": {
+            "[:then, 34, 359, 6, 359, 37]": 0,
+            "[:else, 35, 359, 6, 359, 57]": 0
+          }
+        }
+      }
+    },
+    "timestamp": 1732379975
```

```suggestion
    "timestamp": 1732380988
```



<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>